src/App.js

```diff
@@ -92,13 +92,21 @@ class App extends Component {
   }
 
   render() {
-    return (
-      <div>
-        <Header user={this.state.user} />
-        <img src={logo} alt="logo" />
-        <Footer />
-      </div>
-    );
+    const isLoggedIn = Object.keys(this.state.user).length;
```

Comment:
I prefer something like this:

```suggestion
    const isLoggedIn = !!this.state.user;
```

In my opinion, having user = null when the user is not authenticated, is semantically more correct.

Reply:
thanks!